osu.Game/Overlays/OverflowScrollingContainer.cs

```diff
@@ -0,0 +1,87 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using System;
+using osu.Framework.Allocation;
+using osu.Framework.Bindables;
+using osu.Framework.Graphics;
+using osu.Framework.Graphics.Containers;
+using osuTK;
+
+namespace osu.Game.Overlays
+{
+    public partial class OverflowScrollingContainer : CompositeDrawable
```

Comment:
Can we call this `MarqueeContainer` or something similar? I was very confused when looking at this in review phase (i expected there to be a nested `ScrollContainer` 😅)

Reply:
Sure. I like "marquee" as a name, bit of a throwback 😄